src/zope/app/container/browser/tests/test_contents_functional.py

```diff
@@ -348,15 +350,15 @@ def test_copy_then_delete_with_unicode_name(self):
             basic='mgr:mgrpw',
             form={
                 'ids:list' : u'voil\xe0'.encode('utf-8'),
```

Comment:
That whole line did change, from `'ids' : (u'voil\xe0',),` (with space) to `'ids:list': u'voil\xe0'.encode('utf-8'),` (still with space). But I'll take care of it before merge.